wp-content/themes/mrb-bct/page-client.php

```diff
@@ -0,0 +1,183 @@
+<?php 
+session_start();
+
+/* Template Name: Client Listing */ 
```

Comment:
This appears to be admin level content. So information for admins of the site only?

I'm interested to learn what made you choose to present this on the front end (as opposed to within `wp-admin` dashboard? 

What other options might be available to you?